src/script.h

```diff
@@ -72,6 +72,17 @@ typedef struct scriptFlag {
 
 extern scriptFlag scripts_flags_def[];
 
+/* Lua context */
+typedef struct luaCtx {
+    lua_State *lua; /* The Lua interpreter. We use just one for all clients */
+    client *lua_client;   /* The "fake client" to query Redis from Lua */
+    dict *lua_scripts;         /* A dictionary of SHA1 -> Lua scripts */
+    list *lua_scripts_lru_list; /* A list of SHA1, first in first out LRU eviction. */
+    unsigned long long lua_scripts_mem;  /* Cached scripts' memory + oh */
+} luaCtx;
+
+extern struct luaCtx lctx;
```

Comment:
what about adding a new interface that just retures `lua_scripts`.

Reply:
Thanks for the suggestion. This is my first time submitting a PR, so I'm a bit lost.
I've thought about this too. Which header file should the function declaration go in? Script.h or server.h?

Reply:
in script.h